### PR TITLE
Added 2 Pingdom templates

### PR DESCRIPTION
**Pull request by sarbis**
_pingdom-pause-resume-check.json_  is kind of duplicating functionality of _pause-resume-pingdom-check.json_ but follows with template contributing guidelines.

### CODE REVIEW

File: step-templates/pingdom-create-uptime-check.json [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
...ouldcontain\", $pingdomCheckShouldContain)\n    }\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckShouldNotContain)) {\n        $apiParameters.Add(\"shouldnotcontain\", $pingdomCheckShouldNotContain)\n    }\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckPostData)) {\n        $apiParameters.Add(\"postdata\", $pingdomCheckPostData)\n    }\n\n    If ($apiParameters.Count -gt 0) {\n        $queryString = \"\"\n        $apiParameters.Keys | ForEach-Object { \n            $queryString += ($_ + \"=\" + [Web.HttpUtility]::HtmlEncode($apiParameters.Item($_)) + \"&\")  \n        }\n        $queryString = $queryString.Substring(0, $queryString.Length - 1)\n        $url += \"?$queryString\"\n    }\n\n    Write-Host \"Creating new uptime check: $url\"\n    Try {\n        $response = Invoke-RestMethod -Uri $url -Method Post -ContentType \"application/json\" -Credential $credential -Headers $headers\n        Write-Host \"Successfully added uptime check in Pingdom: Name = \" + ($response.check.name) + \", Id = \" + ($response.check.id)\n    } Catch {\n        Write-Error \"Error occured when adding uptime check in Pingdom: \" $_\n    }\n\n    Write-Host \"End PingdomCreateUptimeCheck\"\n}",
```
Comment (alfhenrik) [inline step-templates/pingdom-create-uptime-check.json]: The function `Test-Any` isn't called anywhere (that I can see), can you please remove it

File: step-templates/pingdom-pause-resume-check.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
...omCheckId = $checkFiltered.id\n    }\n\n    If ([string]::IsNullOrEmpty($pingdomCheckId)) {\n        Write-Warning \"Check with name $pingdomCheckName was not found!\"\n        Exit\n    }\n\n    # Pause or resume check\n\n    $url += \"/$pingdomCheckId\"\n\n    $apiParameters = @{}\n    If ($pingdomCheckPaused -eq \"True\") {\n        $apiParameters.Add(\"paused\", \"true\")\n    } Else {\n        $apiParameters.Add(\"paused\", \"false\")\n    }\n\n    If ($apiParameters.Count -gt 0) {\n        $queryString = \"\"\n        $apiParameters.Keys | ForEach-Object { \n            $queryString += ($_ + \"=\" + [Web.HttpUtility]::HtmlEncode($apiParameters.Item($_)) + \"&\")  \n        }\n        $queryString = $queryString.Substring(0, $queryString.Length - 1)\n        $url += \"?$queryString\"\n    }\n\n    Write-Host \"Modifying uptime check: $url\"\n    Try {\n        $response = Invoke-RestMethod -Uri $url -Method Put -ContentType \"application/json\" -Credential $credential -Headers $headers\n        Write-Host $response.message\n    } Catch {\n        Write-Error \"Error occured when adding uptime check in Pingdom: \" $_\n    }\n\n    Write-Host \"End PingdomPauseResumeCheck\"\n}",
```
Comment (alfhenrik) [inline step-templates/pingdom-pause-resume-check.json]: The function `Test-Any` isn't called anywhere (that I can see), can you please remove it

**Comment by alfhenrik** [comment]
Hi @sarbis,

Thank you for your contribution!

I've reviewed the two step templates and have left some comments for you. If you could also sign the CLA that'd be great!

Thank you and kind regards,
Henrik

File: step-templates/pingdom-create-uptime-check.json (hunk 1 above)
Comment (sarbis) [inline step-templates/pingdom-create-uptime-check.json]: It is actually called:
`$checkExists = $response.checks | Where-Object { $_.name -eq $pingdomCheckName } | Test-Any`

**Comment by sarbis** [comment]
Hi @alfhenrik !

Sorry for messing up with commits here, but tried to fix all of your requested changes.

File: step-templates/pingdom-create-uptime-check.json [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+  "Id": "70ea4820-7d02-4015-a691-8c77b5ab14d5",
+  "Name": "Pingdom - Create Uptime Check",
+  "Description": "Creates Pingdom http check using [Create New Check API method](https://www.pingdom.com/resources/api#MethodCreate+New+Check).",
+  "ActionType": "Octopus.Script",
+  "Version": 27,
+  "CommunityActionTemplateId": null,
+  "Properties": {
+    "Octopus.Action.Script.ScriptSource": "Package",
+    "Octopus.Action.RunOnServer": "false",
+    "Octopus.Action.Script.ScriptFileName": "#{Utilities.PingdomCreateUptimeCheckScriptPath}",
```
Comment (alfhenrik) [inline step-templates/pingdom-create-uptime-check.json]: This should probably use the inline script option 😃

Comment (sarbis) [inline step-templates/pingdom-create-uptime-check.json]: Oh boy, forgot to change it. :D 
That's how we reference scripts in our own templates.

**Comment by sarbis** [comment]
Should be ok now.

**Pull request merged**